ui/public/i18n/language.json

```diff
@@ -607,8 +607,8 @@
     "role_updated_error": "Role update errors",
     "role_deleted_ok": "Role deleted OK",
     "role_deleted_error": "Role deleted errors",
-    "define_authorization": "Define authorizations for this group",
-    "define_authorization_description": "Select views and applications that members of this group can view.",
+    "define_authorization": "Define the authorizations for this group",
+    "define_authorization_description": " Select the System views and Applications that members of this group can manage",
```

Comment:
There is an empty space in ` Select the System views...`

Reply:
I'm sending a patch